en/user-guide/index.rst

```diff
@@ -50,40 +91,57 @@ To use this you have to `enable mock locations on your Android device
 <https://www.youtube.com/watch?v=v1eRHmMiRJQ>`_.
 
 
-Digitize
+Digitalizing
```

Comment:
```suggestion
Digitzing
```

The QGIS documentation uses digitize consistently, I think we should stick to that.